<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Introduction</class>
 <widget class="QMainWindow" name="Introduction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 控制滚动条*/
QScrollBar:vertical{
	background-color:transparent;
	width:10px;
	height:200px;
	padding-top:10px;
	padding-bottom:10px;
}  
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical{
	background-color:rgb(194,137,104);
}        
QScrollBar::handle:vertical{
	background:rgba(124,64,44,0.8);
}   
QScrollBar::handle:vertical:hover,QScrollBar::handle:vertical:pressed{
	background:rgba(104,54,34,0.8);
    border:0px solid rgb(100,100,100);
}

QScrollBar::sub-line:vertical{
	height:20px;
	width:20px;
	border:0px solid rgb(128,217,197);
	background:rgba(159,100,74,1) url(:/image/icon/toolIcon/icon_up.png);
}
QScrollBar::add-line:vertical{
	height:20px;
	width:20px;
	border:0px solid rgb(128,217,197);
	background:rgba(159,100,74,1) url(:/image/icon/toolIcon/icon_down.png);
}


QScrollBar:horizontal{
	background-color:transparent;
	width:140px;
	height:0px;
	padding-left:12px;
	padding-right:12px;
}  
QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal{
	background-color:transparent;
}        
QScrollBar::handle:horizontal{
	background:rgba(41,43,45,0.8);
	border:0px solid rgba(21,23,25,1);
}    
QScrollBar::handle:horizontal:hover,QScrollBar::handle:horizontal:pressed{					  		      
    background:rgba(31,33,35,1);
	border:0px solid rgb(255,255,255);
}

/* 控制按钮图片*/


QToolButton#btnReturn{
	border-image:url(:/resources/image/btnReturn.png);
}
QToolButton#btnReturn:hover{
	border-image:url(:/resources/image/btnReturn1.png);
}


QToolButton#btnExchange{
	border-image:url(:/resources/image/btnExchange.png);
}
QToolButton#btnExchange:hover{
	border-image:url(:/resources/image/btnExchange1.png);
}

QToolButton#btnSave{
	border-image:url(:/resources/image/btnSave.png);
}
QToolButton#btnSave:hover{
	border-image:url(:/resources/image/btnSave1.png);
}

QToolButton#btnLast{
	border-image:url(:/resources/image/btnLast.png);
}
QToolButton#btnLast:hover{
	border-image:url(:/resources/image/btnLast1.png);
}

QToolButton#btnNext{
	border-image:url(:/resources/image/btnNext.png);
}
QToolButton#btnNext:hover{
	border-image:url(:/resources/image/btnNext1.png);
}

QLabel#chessbookLabel{
	image:url(:/resources/image/chessbookLabel.png);
}

QWidget#chessbookWidget{
	border-radius:4px;
	border:4px solid rgb(219,149,96);
}

#chessbookEdit{
	border:0px;
	background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(144, 80, 45, 255), stop:1 rgba(173, 104, 62, 255));
}

QLabel#boardInfoLabel{
	image:url(:/resources/image/rules.png);
}

QWidget#boardInfoWidget{
	border-radius:4px;
	border:4px solid rgb(219,149,96);
}

QTextEdit#boardInfoEdit{
	border:0px;
	background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(144, 80, 45, 255), stop:1 rgba(173, 104, 62, 255));
}





</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="BoardView" name="boardView" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>420</width>
      <height>420</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="controlView" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>311</width>
      <height>451</height>
     </rect>
    </property>
    <widget class="QWidget" name="controlWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QToolButton" name="btnLast">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="btnNext">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chessbookWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>311</height>
      </rect>
     </property>
     <widget class="QLabel" name="chessbookLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QListWidget" name="chessbookEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>256</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="btnReturn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QWidget" name="boardInfoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>171</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QLabel" name="boardInfoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="boardInfoEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 宋体 Std L</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Adobe 宋体 Std L'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoardView</class>
   <extends>QWidget</extends>
   <header location="global">BoardView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
